test/new-e2e/tests/installer/docker.go

```diff
@@ -1,132 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
-
-// Package updater contains tests for the updater package
-package updater
-
-import (
-	"fmt"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
-	"github.com/DataDog/test-infra-definitions/components/os"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-// installDocker installs docker on the host
-func installDocker(distro os.Descriptor, arch os.Architecture, t *testing.T, host *components.RemoteHost) {
```

Comment:
same for docker, if you remove we'll need to rewrite those tests to cover the injector install section https://github.com/DataDog/datadog-agent/blob/main/pkg/fleet/installer/service/docker.go